### ISSUE TITLE
Most Settings unavailable in version 0.7.0.0

### ISSUE THREAD

**Issue by rashil2000** [duke7553/files-uwp#424]
**Describe the bug**
Most options in the new Settings page are greyed-out, i.e. unavailable.

**To Reproduce**
1. Install the latest pre-release using the Install.ps1 script.
2. Open Files and grant File Access permission.
3. Open File->Settings.
4. Go to On Startup tab. Only the first option 'Open a new tab' is available. 'Continue where you left off' and 'Open a specific page or pages' are unavailable.
5. Go to Preferences tab. The 'Pin Onedrive to the sidebar' option is unavailable. Also, clicking on 'Replace common locations' button breaks the app and makes it unusable. I had to reinstall it. 
6. Go to Files and Folders tab. Here, all the options are unavailable. Refer the screenshot below. 

**Screenshot**
![Screenshot (135)](https://user-images.githubusercontent.com/46838874/75091121-f5c16d00-558f-11ea-87e6-406636a1bc56.png)

**Desktop:**
 - Windows 10 Home Version 1909 (OS Build 18363.657)
 - Files UWP Version 0.7.0.0

**Comment by yaichenbaum** [comment]
The options that are grayed out are ones that have not been implemented but are coming in the future.
Can you show a screenshot of how the pin to OneDrive option looks?

**Comment by rashil2000** [comment]
![Screenshot (137)](https://user-images.githubusercontent.com/46838874/75103645-7da48700-5623-11ea-8092-65ea34953565.png)
All three options here have some problem. The first one is greyed out, the second breaks the app as I mentioned earlier, and the third, after uncommenting the PowerShell Core 6 part and replacing it with `powershell.exe` as I only have the legacy PowerShell, the terminal opens initially with a blank screen and then hangs up.

**Comment by lampenlampen** [comment]
@rashil2000 For the legacy powershell you need a different argument:

```
-noexit -command "cd '{0}'"
```

**Comment by rashil2000** [comment]
Oh, thanks for that! 
Also, could you please include the instructions in the Readme file as to how to build the app from source?

**Comment by yaichenbaum** [comment]
@rashil2000 There is an open issue for that already 🙂
Its something we plan on doing in the next few days.

**Comment by lampenlampen** [comment]
In Short:
1. Install Visaul Studio 2019 & UWP Development Kit [link](https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019).
2. Clone the source and open the `Files.sln` in VS.
3. VS installs all missing dependencies for you.
3. Launch the package project.

PS: I don't recommend changing the install path of Visual Studio.

**Comment by rashil2000** [comment]
That's exactly the problem I have. The UWP Development Workload costs around 10GB, but I only have around 15GB available in my SSD (C drive). I'm looking for workarounds though.

**Comment by lampenlampen** [comment]
So i changed my normal installation path and managed to get it to work 👍.
The only problem I had, was that VS couldn't find the Desktop Expansion SDK.
On some microsoft post someone suggested to copy the expansion sdk to the normal folder and than it worked.

**Comment by yaichenbaum** [comment]
This should be fixed in v0.7.1. The update is currently in certification in the Microsoft Store and assuming nothing goes wrong, it should roll out sometime tomorrow.

**Comment by yaichenbaum** [comment]
It seems that this issue carried over to v0.7.1, we have a new fix in the works that will be included with v0.7.2.

**Comment by yaichenbaum** [comment]
This should be fixed with the [v0.7.2](https://github.com/duke7553/files-uwp/releases/tag/v0.7.2) release, which is currently in certification in the Microsoft Store.

**Comment by rashil2000** [comment]
Looks like many of them are still unavailable.

**Comment by yaichenbaum** [comment]
@rashil2000 Can you clarify what you mean by unavailable?

**Comment by rashil2000** [comment]
The options are greyed out.

![Screenshot (189)](https://user-images.githubusercontent.com/46838874/77336996-51794480-6d4e-11ea-9e41-627b25e17498.png)
![Screenshot (191)](https://user-images.githubusercontent.com/46838874/77337002-53430800-6d4e-11ea-95cd-b81d29618fdb.png)
![Screenshot (190)](https://user-images.githubusercontent.com/46838874/77337004-53db9e80-6d4e-11ea-9d84-03bc5ecdefad.png)

**Comment by yaichenbaum** [comment]
The options that are grayed out are ones that have not been implemented but are coming in the future.
I am not sure why the OneDrive option is not enabled for you but I will do some investigations.

**Comment by rashil2000** [comment]
Okay. 
By the way I have the OneDrive app installed too.

**Comment by yaichenbaum** [comment]
It shouldn't make a difference, the only thing that toggle does is remove the OneDrive option from the sidebar.

**Comment by rashil2000** [comment]
I noticed very strange behavior here. 
I reinstalled the app. The OneDrive option was already there this time. Clicking on it just crashed the app. 
Also, when I switched the Date format in Settings to System, the OneDrive sidebar option disappeared. 😐

**Comment by yaichenbaum** [comment]
@rashil2000 Would you be able to get information from the log file so we can investigate this?

**Comment by duke7553** [comment]
@rashil2000 Do you have the classic OneDrive app installed or the Microsoft Store one?

**Comment by rashil2000** [comment]
@rashil2000 Sure, I'll be happy to help. I just don't know how to get those logs. Can you tell how?

@duke7553 I think I have the Store version, since in the Store I can see it installed, and in the Control Panel I can't see it. Is there any other way to check?

**Comment by yaichenbaum** [comment]
You can get the log file by going to the folder %localappdata%/Packages, search there for the folder Files and there in the folder LocalState should be a file debug.txt, which contains a stacktrace.

**Comment by duke7553** [comment]
@rashil2000 Yep. Our app is only compatible with the version of OneDrive that ships with Windows instead of the Store app. 

Please download this here: https://products.office.com/en-us/onedrive/download

**Comment by duke7553** [comment]
If you want to see if the correct version of OneDrive is installed, check the system tray for the icon
![image](https://user-images.githubusercontent.com/20365014/77446715-2120bd00-6dc5-11ea-8767-a857c4236046.png)

**Comment by rashil2000** [comment]
`
2020-03-24 22:29:02.1329|INFO|Files.App|App launched
2020-03-24 22:29:04.2303|ERROR|Files.App|Value cannot be null.
Parameter name: path
2020-03-24 22:30:27.5235|INFO|Files.App|App launched
2020-03-24 22:30:45.3662|INFO|Files.App|App launched
2020-03-24 22:31:22.9186|INFO|Files.App|App launched
2020-03-24 22:32:05.4078|INFO|Files.App|App launched
2020-03-24 22:32:19.5099|INFO|Files.App|App launched
2020-03-24 22:32:50.4351|INFO|Files.App|App launched
2020-03-24 22:33:08.5943|INFO|Files.Filesystem.DrivesManager|Drive added: C:\, Fixed
2020-03-24 22:33:08.6132|INFO|Files.Filesystem.DrivesManager|Drive added: D:\, Fixed
2020-03-24 22:33:08.6332|INFO|Files.Filesystem.DrivesManager|Drive added: E:\, Fixed
2020-03-24 22:33:08.6481|INFO|Files.App|App launched
2020-03-24 22:33:29.1000|INFO|Files.Filesystem.DrivesManager|Drive added: C:\, Fixed
2020-03-24 22:33:29.1180|INFO|Files.Filesystem.DrivesManager|Drive added: D:\, Fixed
2020-03-24 22:33:29.1409|INFO|Files.Filesystem.DrivesManager|Drive added: E:\, Fixed
2020-03-24 22:33:29.1559|INFO|Files.App|App launched
`
@yair100 Here's the log file. 
At `22:32:05.4078`, I opened Settings, but didn't change anything. Then I opened a new tab (without exiting). In the new tab, the OneDrive option was gone. Then at `22:32:19.5099`, I reopened the app. The OneDrive option was still not there.
@duke7553 Oh, I was just using the version that came with my Windows 10 installation. It was the Store version.
